app/javascript/controllers/new_message_controller.js

```diff
@@ -27,8 +27,10 @@ export default class extends Controller {
   }
 
   submitForm() {
-    if (this.cleanInputValue.length > 0)
-      this.element.submit()
+    if (this.cleanInputValue.length > 0) {
+      this.element.requestSubmit()
+      this.inputTarget.value = ''
```

Comment:
Every so often I'm able to submit a message and the form input doesn't clear itself. I don't know why but to fix it I just always cleared it. I added a test for this as well.